WordPress/src/main/java/org/wordpress/android/ui/reader/services/update/ReaderUpdateLogic.java

```diff
@@ -176,11 +177,10 @@ public void run() {
                 localTopics.addAll(ReaderTagTable.getFollowedTags());
                 localTopics.addAll(ReaderTagTable.getBookmarkTags());
                 localTopics.addAll(ReaderTagTable.getCustomListTags());
+                localTopics.addAll(ReaderTagTable.getDiscoverPostCardsTags());
 
-                if (
-                        !localTopics.isSameList(serverTopics)
-                        || displayNameUpdateWasNeeded
-                ) {
+                boolean didChangeFollowedTags = false;
+                if (!localTopics.isSameList(serverTopics)) {
                     AppLog.d(AppLog.T.READER, "reader service > followed topics changed "
                                               + "updatedDisplayNames [" + displayNameUpdateWasNeeded + "]");
```

Comment:
Since the variable `displayNameUpdateWasNeeded` was removed from the `if` and is now only used in this `AppLog`, would it make sense to remove it? If so, perhaps we can even remove the `displayNameUpdateWasNeeded` method.

Reply:
Good point, I thought I had removed it but I think I forgot! Thanks for noticing that!